src/js/controller/Setup.js

```diff
@@ -62,11 +65,18 @@ define([
         ];
 
         this.start = function () {
+            _setupFailureTimeout = setTimeout(_setupTimeoutHandler.bind(this), _errorTimeoutDelay);
+
             _.defer(_nextTask);
         };
 
+        function _setupTimeoutHandler(){
+            this.destroy();
+            _error('Setup Timeout Error: Setup took longer than '+_errorTimeoutDelay+' seconds to complete.');
```

Comment:
That's going to read: "...Setup took longer than 10000 seconds to complete." :eyes: 

Make `_errorTimeoutDelay = 10` and call `setTimeout(fn, _errorTimeoutDelay * 1000)`